# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/update-angle.py
# Commit: 3064469c073d
# Full Hash: 3064469c073d1f789c2a82f76c7af1edbcd1334f
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 1
# Description:
#   Bug 1578576 - Part 1: Update angle updater and merge base r=jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44561
# ==============================================================================

diff -r 7a3378adb340 -r 3064469c073d gfx/angle/update-angle.py
--- a/gfx/angle/update-angle.py	Sat Sep 07 18:17:45 2019 +0000
+++ b/gfx/angle/update-angle.py	Sat Sep 07 13:36:46 2019 +0000
@@ -339,6 +339,7 @@
     return
 
 REGISTERED_DEFINES = {
+    'ANGLE_CAPTURE_ENABLED': True,
     'ANGLE_EGL_LIBRARY_NAME': False,
     'ANGLE_ENABLE_D3D11': True,
     'ANGLE_ENABLE_D3D9': True,
